src/ploigos_step_runner/step_implementers/push_artifacts/npm_push_artifacts.py

```diff
@@ -0,0 +1,152 @@
+"""`StepImplementer` for the `package` step using Maven.
+
+Step Configuration
+------------------
+Step configuration expected as input to this step.
+Could come from:
+
+  * static configuration
+  * runtime configuration
+  * previous step results
+
+Configuration Key            | Required? | Default | Description
+-----------------------------|-----------|---------|-----------
+`pom-file`                   | Yes       | `'pom.xml'` | pom used when executing maven.
+`tls-verify`                 | No        | `True`  | Disables TLS Verification if set to False
+`maven-profiles`             | No        | `[]`    | List of maven profiles to use.
```

Comment:
@dwinchell a bunch of params that all say `maven-*`. i assume copy paste issue.

Reply:
yep. good catch